assets/css/pa-messages.module.scss

```diff
@@ -0,0 +1,204 @@
+@import "./variables.scss";
+
+@mixin flat-button {
+  --bs-btn-active-color: var(--bs-btn-color);
+  --bs-btn-hover-color: var(--bs-btn-color);
+  --bs-btn-active-bg: var(--bs-btn-bg);
+  --bs-btn-hover-bg: var(--bs-btn-bg);
+  --bs-btn-active-border-color: var(--bs-btn-border-color);
+  --bs-btn-hover-border-color: var(--bs-btn-border-color);
+}
+
+.editPage {
```

Comment:
If we'd like to keep using `kebab-case` within CSS, but use the more convenient `camelCase` when referencing styles in JS, it looks like css-loader has an [exportLocalsConvention](https://webpack.js.org/loaders/css-loader/#exportlocalsconvention) option that can accommodate this. I'd slightly prefer this, especially given we're still using kebab-case names for mixins and variables. But it may also be convenient to have the names match up exactly for cross-referencing.

Reply:
Interesting. I'd be hesitant to pay the cognitive overhead of having to translate between the two, especially since doing so would make grep-based exploration of the code more difficult. I'm curious to head other opinions on this.

As for the generated names, yes, we can tweak that so it's more amenable to debugging.